### DISCUSSION TITLE
Selecting multiple items with different lengths

### DISCUSSION THREAD

**Discussion by c0de-intention** [helix-editor/helix#5493] [Q&A] [answered]
I had a situation where I wanted to selected multiple items so I could use the match to add surrounding quotation marks to items with different lengths. 
For example:
```ts
const myValues = [
   value1,
   longerValue2,
   val3
]
```

I wanted to get it to the following:

```ts
const myValues = [
   'value1',
   'longerValue2',
   'val3'
]
```
I ended selected one by one with `w` and then `m s '`.
I was wondering if there was a quicker/smarter way to do this

**Comment by gabydd** [comment, marked as answer]
if you start on the v of value(``fv`` from the first line) you can do ``CCwms'`` C makes a new selection under, then you can select the word and surround all of them with ``'``

**Reply by c0de-intention** [reply]
Thanks @gabydd I was just coming back to add that, as i ran into it again
